File: java/Level A/6 - A. Gravity Flip.java
import java.util.Arrays;
import java.util.Scanner;

public class Main {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        int size = scanner.nextInt(); // 4
        int [] arr = new int[size]; // [    ]
        for (int i=0;i<size;i++){ // 0 1 2 3
            arr[i] = scanner.nextInt();
        }
        Arrays.sort(arr);
        for (int i=0;i<size;i++){ // 0 1 2 3
            System.out.print(arr[i] + " ");
        }
    }
}

File: c++/Level A/6- A. Gravity Flip.cpp
#include <bits/stdc++.h>
using namespace std;
int main(){// 1   20
	int size;// 4
	cin>>size;
	
	int arr[size];
	for(int i=0;i<size;i++){// 0 1 2 3
		cin>>arr[i]; // [3,2,1,0]   [0 1 2 3]
	}
	sort(arr,arr+size);
	for(int i=0;i<size;i++){// 0 1 2 3
		cout<<arr[i]<<" "; // [3,2,1,0]   [0 1 2 3]
	}
}

